fulp_modules/features/antagonists/bloodsuckers/code/bloodsucker/bloodsucker_datum.dm

```diff
@@ -267,6 +272,23 @@
 		old_body.remove_traits(bloodsucker_traits, BLOODSUCKER_TRAIT)
 	new_body.add_traits(bloodsucker_traits, BLOODSUCKER_TRAIT)
 
+	//Give the datum the blood volume of its new body.
+	bloodsucker_blood_volume = new_body.blood_volume
```

Comment:
wtf..

Reply:
Bloodsucker blood volume and normal blood volume are 2 separate things, we set the blood of the body to that of the bloodsucker (I believe somewhere in the life tick) for health analyzers and so masquerade could fake it, we never care about the actual blood of the corpse itself. This doesn't *really* matter and I wouldn't care if we do this.

Reply:
I understand what you mean. Still, we'd only be doing this _once_ per body transfer. It allows Bloodsuckers with zero blood to be safely transferred to a new body without anyone having to worry about them instantly entering Frenzy. It also just gives them a fresher slate overall, and might be more in line with what the average player would intuitively expect.

Reply:
Sounds reasonable then, didn't think about its effects w/ Frenzy.